sql/migrate/dir.go

```diff
@@ -179,6 +179,37 @@ func (f LocalFile) Bytes() []byte {
 	return f.b
 }
 
+// Directive returns the (global) file directives that match the provided name.
+// File directives are located at the top of the file and should not be associated with any
+// statement. Hence, double new lines are used to separate file directives from its content.
```

Comment:
I am no fan of different meanings for \n and \n\n. is there another solution we can think of?